src/networkProviders/apiNetworkProvider.ts

```diff
@@ -34,7 +34,9 @@ export class ApiNetworkProvider implements INetworkProvider {
         this.config = { ...defaultAxiosConfig, ...config };
         this.backingProxyNetworkProvider = new ProxyNetworkProvider(url, proxyConfig);
         this.axios = getAxios();
-        extendUserAgent(this.userAgentPrefix, this.config);
+        if (isBackend()) {
```

Comment:
We can also hide `if !isBackend()` within `extendUserAgent()` or  `extendUserAgentIfAppropriate()`.